erdpy_network/transaction_completion_strategy.py

```diff
@@ -16,6 +27,36 @@ def is_completed(self, transaction: ITransactionOnNetwork) -> bool:
         return not transaction.get_status().is_pending()
 
 
+# this class is similar to the one in erdjs-network-providers
```

Comment:
Perhaps add a link towards the erdjs implementation?

Reply:
Added a link to the file.